modules/policy-criteria.adoc

```diff
@@ -274,11 +274,19 @@ AND, OR
 6+| *Section: Container configuration*
 
 | Environment Variable
-| Check environment variables by name or value.
+| Check environment variables by name or value. When you create a policy that includes the environment variable attribute,
+you can choose which types of environment variables the policy should match. For example, you can specify raw values, which are provided directly in the deployment YAML, or you can specify references to values from ConfigMaps, Secrets, fields, or resource requests or limits.
+For any type other than a raw value specified directly in the deployment YAML, the corresponding `value` attribute of the policy rule is ignored. In this case, the policy match is evaluated on the existence of the specified environment variable type. Additionally, this criteria disallows the creation of policies with a non-empty `value` attribute for types other than raw values.
 | Environment Variable
-| RAW=key=value to match an environment variable as directly specified in the deployment configuration with a specific key and value. `value` can be omitted to match on only the key. +
+a| RAW=key=value to match an environment variable as directly specified in the deployment configuration with a specific key and value. `value` can be omitted to match on only the key. +
+
+If the environment variable is not directly defined in the configuration, then the format SOURCE=KEY can be used, where SOURCE is one of the following objects. In this list, the API object label is provided first; the label in the user interface is provided in parentheses.
```

Comment:
I'd add the list after: following objects.  And then continue "In the above list, the API object label ...." after the bulleted list.